src/vm/i386/jitinterfacex86.cpp

```diff
@@ -1397,10 +1397,16 @@ void EmitFastGetSharedStaticBase(CPUSTUBLINKER *psl, CodeLabel *init, bool bCCto
         // DoInit:
         psl->EmitLabel(DoInit);
 
-#if defined(UNIX_X86_ABI)
-        // sub esp, 8 ; to align the stack
-        psl->X86EmitSubEsp(8);
-#endif
+#ifdef FEATURE_PAL
```

Comment:
I though that this minimal frame is required even when 16-byte alignment is not required (``UNIX_X86_ABI`` currently means).